docs/attributes-registry/README.md

```diff
@@ -0,0 +1,27 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Registry
+weight: -2
+--->
+
+# Attributes Registry
+
+The attributes registry is the place where attributes are defined. An attribute definition covers the following properties of an attribute:
+
+- the `id` (the fully qualified name) of the attribute
+- the `type` of the attribute
+- a `brief` description of the attribute and optionally a longer `note`
+- example values
+
+Attributes defined in the registry can be used in different semantic conventions. Attributes should be included in this registry before they are used in semantic conventions. Semantic conventions may override all the properties of an attribute except for the `name` and `type` in case it's required for a particular context. In addition, semantic conventions specify the requirement level of an attribute in the corresponding context.
+
+A definition of an attribute in the attributes registry does not necessarily imply that this attribute is used in any semantic conventions, yet.
+
+All registered attributes are listed by namespace in this registry.
+
+> **Warning** The following registry overview is not complete, yet.
```

Comment:
```suggestion
> **Warning**
> The following registry overview is not complete, yet.
```

If you want to have the nice icon show up the warning text needs to be alone in the line